### DISCUSSION TITLE
Bad lingature for coding font on doc page

### DISCUSSION THREAD

**Discussion by amio** [pmndrs/jotai#2329] [Bug report]
## Link to reproduction

For example https://jotai.org/docs/basics/concepts

![image](https://github.com/pmndrs/jotai/assets/215282/10493a8f-9da2-41e4-90de-366acd5df2bd)

Also notice the `fi` in `filter()`.

It would be nicer to use a font without lingature, like menlo:

![image](https://github.com/pmndrs/jotai/assets/215282/aa27024e-452d-485c-97b2-a6cd933c9313)

**Comment by dai-shi** [comment]
Thanks for reporting!
I'd leave it to @sandren , but maybe a PR would be welcome.
